full-stack-tests/presentation/src/performance/DataViz.test.ts

```diff
@@ -0,0 +1,625 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/* eslint-disable no-console */
+import { expect } from "chai";
+import { assert, Guid, Id64String, OrderedId64Iterable, StopWatch } from "@itwin/core-bentley";
+import { QueryBinder, QueryRowFormat } from "@itwin/core-common";
+import { IModelConnection, SnapshotConnection } from "@itwin/core-frontend";
+import {
+  ChildNodeSpecificationTypes, ClassInfo, ContentSpecificationTypes, DefaultContentDisplayTypes, Descriptor, Field, InstanceKey, KeySet, NodeKey,
+  PropertiesField, PropertiesFieldDescriptor, PropertyInfo, RelationshipDirection, Ruleset, RuleTypes, StrippedRelationshipPath, Value,
+} from "@itwin/presentation-common";
+import { Presentation } from "@itwin/presentation-frontend";
+import { ECClassHierarchy, ECClassInfo } from "../ECClasHierarchy";
+import { initialize, terminate } from "../IntegrationTests";
+import { getFieldsByLabel } from "../Utils";
+
+/**
+ * The below specifies what iModel to use and what Fields (properties) to use for simulating DataViz
+ * component behavior. The fields should be picked based on the iModel.
+ */
+
+// Recommended iModel:
+// https://qa-connect-imodelhubwebsite.bentley.com/Context/892aa2c9-5be8-4865-9f37-7d4c7e75ebbf/iModel/ed0c408b-add2-496e-ac58-5a7e853cbc0b
+const iModelFileName = "S - Bayer - Plantsight connection_dtr-27be16d959645985fca80cff502bd237b82f0f84.bim";
```

Comment:
Is it expected that whoever is running this test will copy this iModel to the assets directory?

Reply:
At the moment - yes. The test is not going to be run as part of the pipeline, so that's not an issue. 

People who may want to run this test locally will probably want to run it against different iModels. I think the most convenient way for that person would be to specify `iTwinId` and `iModelId` and expect the test to download the iModel. However, I'm not sure that's possible due to authorization. And the next most convenient way is to just place an already downloaded iModel into the assets.

Reply:
Good - I mostly wanted to confirm this iModel won't be added to the git repo.